CHANGELOG.md

```diff
@@ -4,6 +4,48 @@ Releases
 This page documents the releases of Sleeper. Performance figures for each release
 are available [here](docs/12-performance-test.md)
 
+## Version 0.18.0
+This contains the following improvements:
+
+Bulk Import:
+- Support use of EMR serverless.
+- Support use of Graviton instances in bulk import EMR.
+- Report on validation status for bulk import jobs.
+- Added option to query rejected jobs in the `IngestJobStatusReport`.
+- Support use of instance fleets in bulk import EMR.
+- Updated default x86 instance types to use m6i equivalents.
+
+Environment:
+- Allow multiple users to access cdk-environment EC2.
+- Allow users to connect to an existing deployed environment without redeploying.
+
+Deployment:
+- Added ability to deploy to multiple subnets.
+- Split properties templating from the deployment process.
```

Comment:
It might be worth noting that this adds the ability to specify your own configuration files while still defaulting to the templates.